docker/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN cd /code && mkdir -p /zipkin-storage-kafka && \
     jar xf *.jar && \
     rm *.jar
 
-FROM openzipkin/zipkin:2.19.1
+FROM openzipkin/zipkin:master
```

Comment:
Seems that `v2.20.2` is not available on docker-hub. It should be updated before merging.

Reply:
@anuraaga hmm this seems to be a pattern

Reply:
Yeah need to take a deep look at this soon, probably next week.